doc-content/enterprise-only/release-notes/rn-whats-new-con.adoc

```diff
@@ -5,180 +5,182 @@ This section highlights new features in {PRODUCT} {PRODUCT_VERSION}.
-    // start node
-    .startNode(1).name("Start").done()
-    // script node in Java language that prints "action"
-    .actionNode(2).name("Action")
-    .action(Dialect.JAVA,
-            "System.out.println(\"Action\");").done()
-    // end node
-    .endNode(3).name("End").done()
-    // connections
-    .connection(1, 2)
-    .connection(2, 3)
-    .build();
-----
+=== Ability to access activity details in boundary events
 
-For more information about process fluent API, see {URL_DEPLOYING_AND_MANAGING_SERVICES}#assembly-kie-apis[_{KIE_APIS}_].
+In the BPMN modeler, you can now set a data output in the *Data Output and Assignments* field for a boundary event. For example, you can set `nodeInstance` as a data output variable for a boundary event, which carries the node instance details to use in a further process when the boundary event is triggered.
+For more information, see {URL_DEVELOPING_PROCESS_SERVICES}#assembly-designing-business-processes[_{DESIGNING_BUSINESS_PROCESSES}_].
 
-=== Enhanced search capabilities
-{PRODUCT} {ENTERPRISE_VERSION} provides enhanced search capabilities, including flexible search criteria for tasks, process instances, and cases. For more information, see {URL_DEVELOPING_PROCESS_SERVICES}#assembly-designing-and-building-cases[_{DESIGNING_CASES}_].
+=== Support for MVEL expressions in data assignments
 
-endif::[]
+You can now add MVEL expressions in the data input and output assignments of a user task. For more information, see {URL_DEVELOPING_PROCESS_SERVICES}#assembly-designing-business-processes[_{DESIGNING_BUSINESS_PROCESSES}_].
 
-== {PLANNER}
+== {PROCESS_ENGINE_CAP}
 
-=== Using {PLANNER} with Spring Boot
+=== {PROCESS_ENGINE_CAP} API improvements
 
-You can follow the example in {URL_DEVELOPING_SOLVERS}#assembly-business-optimizer-springboot[{PLANNER_WITH_SPRING_BOOT}] to create a school timetable.
+The following list provides a summary of the process engine API updates:
 
-=== Employee Rostering rotation pattern redesign
+* You can now use the {PROCESS_ENGINE} API to signal a process instance using its correlation key.
+* You can now use the {PROCESS_ENGINE} API to determine the node type from within an event listener.
+* When using the advanced query feature of the {PROCESS_ENGINE} API, you can now optionally exclude process variables from the search results.
 
-The rotation pattern on the Employee Rostering Rotation window has been redesigned for ease of use. For more information, see {URL_DEVELOPING_SOLVERS}#assembly-optimizer-modifying-ER-template-IDE[_{RUNNING_ROSTER_FOR_PLANNER_IDE}_].
+For more information about using the {PROCESS_ENGINE} API, see {URL_DEPLOYING_AND_MANAGING_SERVICES}#assembly-kie-apis[_{KIE_APIS}_].
 
-== Spring Boot
+=== KIE API improvements
 
-ifdef::PAM[]
+You can now use the KIE REST API to signal a process using its alias. For more information about using the KIE REST API, see {URL_DEPLOYING_AND_MANAGING_SERVICES}#assembly-kie-apis[_{KIE_APIS}_].
```

Comment:
Please confirm with Misha about the link

Reply:
Same as above